src/telegrafs/components/CollectorCard.tsx

```diff
@@ -1,11 +1,21 @@
 // Libraries
-import React, {PureComponent, MouseEvent} from 'react'
+import React, {PureComponent, MouseEvent, RefObject, createRef} from 'react'
 import {connect, ConnectedProps} from 'react-redux'
 import {withRouter, RouteComponentProps, Link} from 'react-router-dom'
 
 // Components
-import {Context} from 'src/clockface'
-import {ResourceCard, IconFont} from '@influxdata/clockface'
+import {
+  ResourceCard,
+  IconFont,
+  FlexBox,
+  ConfirmationButton,
+  ButtonShape,
+  ComponentSize,
+  SquareButton,
+  Popover,
+  Appearance,
+  List,
```

Comment:
good opportunity to alphabetize this list

Reply:
will do !

Reply:
interesting, I made that change and pushed it up but not being refelected in the PR lol

Reply:
![Screen Shot 2021-09-15 at 11 46 40 AM](https://user-images.githubusercontent.com/12561526/133466018-ef195b43-ec61-439f-a7ea-f0681fd55295.png)

Reply:
nvm its showing up now